src/app/components/LiveLabel/index.tsx

```diff
@@ -0,0 +1,53 @@
+/** @jsx jsx */
+/** @jsxRuntime classic */
+import { PropsWithChildren, useContext } from 'react';
+import { jsx } from '@emotion/react';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import styles from './index.styles';
+
+interface LiveLabelProps {
+  service: string;
+  dir?: string;
```

Comment:
This can be deleted - we no require this as a prop, as we get the `dir` from the ServiceContext